app/views/home/_inbox_module.html.erb

```diff
@@ -25,10 +19,16 @@
             <li>
               <%= render 'inbox/reply_button', feedback_comment: inbox_comment.feedback_comment %>
             </li>
+          <li id="read_comment_form_<%= inbox_comment.feedback_comment.id %>">
+            <%= render 'inbox/read_form', inbox_comment: inbox_comment, current_user: current_user %>
+          </li>
+          <li>
+            <%= render 'inbox/delete_form', inbox_comment: inbox_comment, current_user: current_user %>
+          </li>
```

Comment:
It looks like these two buttons used to appear regardless of whether the comment was unreviewed, and now will only appear if the comment isn't unreviewed. Is that an intentional change?

Reply:
Drat. No, it's not! Thanks for catching that. I had to read it three times even after your comment to see where that was happening. /o\ 

So what we want is:
1. Approve, which should only show when the comment is unreviewed
2. Reply, which should only show when the comment is not unreviewed
3. Mark Read, which should appear regardless of reviewed status
4. Delete, which should also appear regardless of reviewed status